### ISSUE TITLE
Drop-down menus on results are not announced as menu items by screen readers

### ISSUE THREAD

**Issue by bojanrajkovic** [Microsoft/workbooks#163]
### Steps to Reproduce

1. Run `2+2` in a cell
2. Open the renderer selection menu next to the result while using a screen reader

Reproduced on Mac, but the same problem exists on Windows.

### Expected Behavior

The screen reader reads the menu items in the correct way (dependent on screen reader--Mac's VoiceOver will differ from Windows' Narrator, or NVDA, or Jaws)

### Actual Behavior

Menu items aren't read out by the screen reader, nor is the selected item indicated in any way.

#### Xamarin Workbooks
| Detail | Value |
| :- | :- |
| Version | 1.4.0-rc1 |
| Git Branch | master |
| Git Hash | 7f3bf81 |
| VSTS Definition | Workbooks & Inspector (Mac) |

#### System Info
| Component | Value |
| :- | :- |
| macOS | Version 10.13.2 (Build 17C88) (64-bit) |
| CPU Cores | 8 / 8 |
| Physical Memory | 16 GB |

#### System-Installed Software
| Component | Version |
| :- | :- |
| Mono | 5.4.1.7 |
| Xamarin Mac | 4.0.0.214 |
| Xamarin iOS | 11.4.0.214 |
| Xamarin Android | 8.1.0-23 |
| Visual Studio | 7.3.0.764 (703000764) |
